docs/docs/getting-started/02-oauth-tutorial.mdx

```diff
@@ -62,13 +82,9 @@ $ openssl rand -base64 32
 
 or https://generate-secret.vercel.app/32 to generate a random value for it.
 
-:::info
-Auth.js is extremely customizable, [our guides section](/guides/overview) will teach you how you can set it up to handle auth in different ways. All the possible configuration options are [listed here](/reference/configuration/auth-config).
-:::
-
-### Exposing the session via provider
+### Exposing the session via `SessionProvider`:
 
-To be able to use `useSession` first you'll need to expose the session context, [`<SessionProvider />`](/reference/react/#sessionprovider), at the top level of your application:
+To use `useSession` first you'll need to expose the session context - [`<SessionProvider />`](/reference/react/#sessionprovider) - at the top level of your application:
```

Comment:
`useSession` wasn't mentioned at this point, we should explain in a sentence what that is.